api/gql_schema.py

```diff
@@ -76,6 +79,43 @@ class Meta:
         fields = "__all__"
 
 
+class CreatePostWithSerializer(SerializerMutation):
```

Comment:
Did not like `SerializerMutation` - too much magic, hard to understand and modify behaviour